roles/network_plugin/calico/tasks/install.yml

```diff
@@ -119,12 +119,13 @@
         -e '/^\s{2,4}annotations:/{:1;/\(devel\)$/!{N;b 1}; /.*/d}'
         {{ local_release_dir }}/calico-{{ calico_version }}-kdd-crds/*.yaml
       when:
-        - calico_version is version('v3.17.0', '<')
+        - calico_version is version('v3.19.0', '<')
```

Comment:
Are you sure about this one ?
I think those annotations are not present in 3.17+

Reply:
Indeed, I looked over the CRDs from 3.17 and 3.18 and they were not there. I just assumed that since there was no other mention of 3.17 before this patch that when condition was an upper bound. That said, the command itself is harmless since it will not change anything in the 3.17+ CRD files but I removed the unnecessary change anyway.

Reply:
I agree it's harmless but in kubespray we tend to carry workaround for way longer than needed, once we remove the 3.17 hash we can also remove this task ;)